vital/tests/test_detected_object.cxx

```diff
@@ -104,3 +104,18 @@ TEST(detected_object, modification)
 
   doby.set_bounding_box( bb );
 }
+
+// ----------------------------------------------------------------------------
+TEST(detected_object, keypoints)
+{
+  kwiver::vital::bounding_box_d bb( 50, 50, 100, 100 );
+  kwiver::vital::detected_object dobj( bb );
+
+  dobj.add_keypoint( "head", kwiver::vital::point_2d( { 11.2, 13.1 } ) );
```

Comment:
Why are you using the initializer list ctor here and not the ctor that takes two doubles?

Reply:
There should be a contructor for that case, but there isn't

Reply:
point_2d isn't Eigen, https://github.com/Kitware/kwiver/blob/master/vital/types/point.h

Reply:
Ah, right, I am mixing that up with `vector_2d`... and that's actually fairly awful that `point` is seriously lacking in useful ctors. @aaron-bray, how horrible do you think it would be to make `point` *inherit* from `Eigen::Matrix` rather than having one as a member? (I believe you can inherit as protected and still use ctor inheritance...)

Alternatively, that said, are we sure we *want* covariance for key points?

Reply:
Short term no, but one could theoretically have uncertain metrics associated with the point locations for these type of keypoints so having them in there isn't an issue